src/css/sideNav.module.css

```diff
@@ -1,12 +1,4 @@
-@keyframes grow {
```

Comment:
This file need to be renamed to `SideNav.module.css`

Reply:
@Bouncey Whoops! Turns out I had an issue I've had before: git isn't case sensitive in its rename detection. This worked locally because I had actually renamed the file, but git didn't count it as a change (and I didn't notice). My previous workaround was to delete, commit, then add back in. I just did a search to see if there was a better way. Turns out you can use `git mv oldname newname` to do this.